### PR TITLE
Add pixel-identity to the list of OpenIddict-based projects maintained by third parties

### PR DESCRIPTION
**Pull request by Nfactor26**
**Description**

**[Pixel Identity](https://github.com/nfactor26/pixel-identity)** is a ready to host OpenId Connect service built on top of OpenIddict and Asp.Net Identity. It provides a blazor UI for managing users, roles, applications and scopes. It also supports multiple databases e.g.  MongoDB, Postgres SQL , Microsoft SQL Server. 

Add the project information on OpenIddict read me page for awareness on this project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,6 +91,7 @@ Instead, **developers are encouraged to execute the conformance tests against th
 - **[OrchardCore OpenID module](https://github.com/OrchardCMS/OrchardCore)**: turnkey OpenID Connect server and token validation solution, built with multitenancy in mind
 - **[OpenIddict UI](https://github.com/thomasduft/openiddict-ui)** by [Thomas Duft](https://github.com/thomasduft): headless UI for managing client applications and scopes
 - **[P41.OpenIddict.CouchDB](https://github.com/panoukos41/couchdb-openiddict)** by [Panos Athanasiou](https://github.com/panoukos41): CouchDB stores for OpenIddict
+- **[pixel-identity](https://github.com/Nfactor26/pixel-identity)** by [Nishant Singh](https://github.com/Nfactor26): Ready to host OpenID Connect service using OpenIddict and Asp.Net Identity with a blazor based UI for managing users, roles, applications and scopes with support for multiple databases.
```
Comment (kevinchalet) [inline README.md]:
```suggestion
- **[pixel-identity](https://github.com/Nfactor26/pixel-identity)** by [Nishant Singh](https://github.com/Nfactor26): Ready to host OpenID Connect service using OpenIddict and ASP.NET Identity with a Blazor-based UI for managing users, roles, applications and scopes with support for multiple databases.
```

**Review by kevinchalet — APPROVED** [review]
Thanks for your PR!

**Comment by Nfactor26** [comment]
wc !! Updated PR as  per suggestion. Looking forward for your feedback and feature suggestions to improve the project.

**Comment by kevinchalet** [comment]
Merged, thanks for your contribution!

Could you please send a PR to update the docs too? https://github.com/openiddict/openiddict-documentation/blob/dev/integrations/index.md

Thanks!

**Pull request merged**